src/app/pages/information/information.page.html

```diff
@@ -0,0 +1,69 @@
+<ion-header *transloco="let t">
```

Comment:
The indentation of `*.html`, `*.ts` and `*.scss` files should be 2 spaces.

Click the button on the toolbar to override the default size.

![Screenshot from 2020-08-17 16-43-03](https://user-images.githubusercontent.com/14951000/90376318-d4f25400-e0a8-11ea-8455-1127b457027b.png)

If you update this feature branch to the latest develop branch, the default indentation size will be updated to 2 spaces in `.vscode/settings.json`.